From 7a829db368f59f27ffdac9879d49c5cf6d3629a8 Mon Sep 17 00:00:00 2001
From: "Kothapeta, BikshapathiX" <bikshapathix.kothapeta@intel.com>
Date: Thu, 24 Aug 2023 12:04:35 +0530
Subject: [PATCH] Fix for GTS checkGpuProfilingRequirements TC

TC failing Error: API level S must support GPU profiling
 graphics.gpu.profiler.support=true

Tracked-On: OAM-104957
Signed-off-by: vdanix <vishwanathx.dani@intel.com>
Signed-off-by: Kothapeta, BikshapathiX <bikshapathix.kothapeta@intel.com>

diff --git a/groups/device-specific/caas/caas.mk b/groups/device-specific/caas/caas.mk
index b577368..1a85e75 100644
--- a/groups/device-specific/caas/caas.mk
+++ b/groups/device-specific/caas/caas.mk
@@ -62,6 +62,10 @@ PRODUCT_LOCALES := en_US en_IN fr_FR it_IT es_ES et_EE de_DE nl_NL cs_CZ pl_PL j
 PRODUCT_AAPT_CONFIG := normal large mdpi
 PRODUCT_AAPT_PREF_CONFIG := mdpi
 
+PRODUCT_VENDOR_PROPERTIES += \
+    graphics.gpu.profiler.support=true
+
+
 PRODUCT_VENDOR_PROPERTIES += \
     ro.soc.manufacturer=$(PRODUCT_MANUFACTURER)
 
-- 
2.42.0

